client_test.go

```diff
@@ -28,7 +28,7 @@ func TestClient_Version(t *testing.T) {
 			gotResp, err := tt.client.GetVersion()
 			require.NoError(t, err)
 			require.NotNil(t, gotResp, "Version() should not return nil value")
-			require.Equal(t, "0.21.0", gotResp.PkgVersion)
+			require.Equal(t, "0.21.1", gotResp.PkgVersion)
```

Comment:
Can you remove this test? We don't want the test to fail when a patch of MeiliSearch is released, like v0.21.1 🙂
@alallema do you agree with this?

Reply:
Done